department-of-veterans-affairs/va.gov-cms
# VACMS-3021: Field Benefit Hub Contacts Cardinality

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3021

## Testing done


## Screenshots


## QA steps

As an editor
- [x] Log into preview environment. (This can be found in devshop) 

- Navigate to the Contact Information paragraph type an confirm the following:
 - [x] Navigate to: `/admin/structure/paragraphs_type/contact_information/fields`
   - Select **Edit** under the Operations column for the **Additional contact** field:
   - Select **Field Settings** tab and confirm the following:
      - [x] Allowed number of values: Limited: 1

- Add content for the following content types to confirm that only a single benefit hub contact is allowed:

## checklist 

- [ ] Navigate to `node/add/checklist`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [ ] Confirm successful save.



## faq_multiple_q_a

- [ ] Navigate to `node/add/faq_multiple_q_a`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.




## step_by_step

- [ ] Navigate to `node/add/step_by_step`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.




## media_list_images

- [ ] Navigate to `node/add/media_list_images`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.




## media_list_videos

- [ ] Navigate to `node/add/media_list_videos`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.





## q_a

- [ ] Navigate to `node/add/manage/q_a`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.





## support_resources_detail_page

- [ ] Navigate to `node/add/support_resources_detail_page`
  - [ ] Scroll down to the bottom and find the 'Contact Information' field section
  - [ ] Select the 'Benefit Hub Contacts' radio button to reveal the 'Add Benefit Hub Contacts' button
  - [ ] Select the 'Add Benefit Hub Contacts' button
  - [ ] Select more than 1 benefit hub contacts from the list and select Add Benefit Hub Contacts
  - [x] Confirm error appears '1 error has been found'
  - [ ] Select only a single benefit hub contacts and select Add Benefit Hub Contacts
  - [x] Confirm successful entry of the select benefit hub within the Benefit Hub Contact section'
  - [ ] Complete remainder of form and select save.
  - [x] Confirm successful save.

## Definition of Done
